docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -861,6 +875,11 @@ KBKDF
         The counter iteration variable will be concatenated after
         the fixed input data.
 
+    .. attribute:: MiddleFixed
```

Comment:
This should have a `.. versionadded` inside the attribute.

Reply:
Thanks, added.